script/Ciel/Day 9/09_01_CIEL09_1C.txt

```diff
@@ -152,17 +167,22 @@ Inside this familiar room, I see, as I expected, a funky doctor, terribly unsuit
 [sha:8febc66f9de91019ac6dd070f2ad9d2c8101e5cb]{
 -- Page 65, Offset 22300.
 -- 「あらぁ？　坊ちゃんったらここの生徒だったのぅ？
-"Oh my? So the young master is a student of this school?
+"What have we here? So this is where the young master goes to school?
 }
 [sha:24efc08ae33ed2ee18aed00853d39bc4b8976c60]{
 -- Page 65, Offset 22301.
 -- アタシびっくりー。もしかしてこれ、運命の赤い糸ってヤツかしらぁ!?　いやーん、こまっちゃう！　これで遠野家の財産独り占めジャーン！
-How shocking. Could this be the red thread of fate?! Oooh, what should I dooo? This means I'll have the fortune of the Tohno family all for myself!
+// Any chance 財産 is supposed to refer to Shiki as an asset of the family?
```

Comment:
Gouto _does_ say something like this but i think the current interpretation is right